gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -238,17 +233,17 @@ private String getVersionName() {
 
   @Override
   public void onGlobalLayout() {
-    if (toolbarWrapper == null || bottomSheetBehavior == null || bottomSheetHeader == null) {
+    if (toolbarWrapper == null || bottomSheetBehavior == null || insets == null) {
       return;
     }
     bottomSheetBehavior.setFitToContents(false);
 
     // When the bottom sheet is expanded, the bottom edge of the header needs to be aligned with
     // the bottom edge of the toolbar (the header slides up under it).
-    bottomSheetBehavior.setExpandedOffset(
-        toolbarWrapper.getHeight() - bottomSheetHeader.getHeight());
+    bottomSheetBehavior.setExpandedOffset(toolbarWrapper.getHeight());
```

Comment:
@dturner The latest version of BottomSheetBehavior now exposes setExpandedOffset, but it must be non-negative. Breaking the header out allows us to set a positive expandedOffset (the height of the scrim for the top inset).